sdk/src/main/java/com/talkable/sdk/models/SharingChannel.java

```diff
@@ -3,13 +3,14 @@
 import com.google.gson.annotations.SerializedName;
 
 public enum SharingChannel {
-    @SerializedName("facebook")         FACEBOOK("facebook"),
-    @SerializedName("facebook_message") FACEBOOK_MESSAGE("facebook_message"),
-    @SerializedName("twitter")          TWITTER("twitter"),
-    @SerializedName("linkedin")         LINKEDIN("linkedin"),
-    @SerializedName("email")            EMAIL("email"),
-    @SerializedName("sms")              SMS("sms"),
-    @SerializedName("other")            OTHER("other");
+    @SerializedName("facebook")             FACEBOOK("facebook"),
+    @SerializedName("facebook_message")     FACEBOOK_MESSAGE("facebook_message"),
+    @SerializedName("twitter")              TWITTER("twitter"),
+    @SerializedName("linkedin")             LINKEDIN("linkedin"),
+    @SerializedName("email")                EMAIL("email"),
+    @SerializedName("sms")                  SMS("sms"),
+    @SerializedName("other")                OTHER("other"),
+    @SerializedName("direct_email_native")  NATIVE_MAIL("direct_email_native");
```

Comment:
I would name it `email_native`

Reply:
@dgtv Too late - this channel is already used in several campaigns, so I had to go with this name